src/smt/process_assertions.cpp

```diff
@@ -219,8 +209,23 @@ bool ProcessAssertions::apply(Assertions& as)
   if (options::solveBVAsInt() != options::SolveBVAsIntMode::OFF)
   {
     d_passes["bv-to-int"]->apply(&assertions);
+    // after running bv-to-int, we need to immediately run
+    // theory-preprocess and ite-removal so that newlly created
+    // terms and assertions are normalized (e.g., div is expanded).
+    d_passes["theory-preprocess"]->apply(&assertions);
+    d_passes["ite-removal"]->apply(&assertions);
```

Comment:
Is this necessary? I thought that ITE removal was subsumed by theory-preprocess.  Can you try removing this line?

Reply:
right!  deleted in https://github.com/CVC4/CVC4/pull/5208/commits/2868064bce934d1641b97c7cbe0ed00bdee45bab